tilequeue/command.py

```diff
@@ -1628,6 +1628,47 @@ def tilequeue_rawr_process(cfg, peripherals):
     rawr_pipeline()
 
 
+def tilequeue_rawr_seed_toi(cfg, peripherals):
+    """command to read the toi and enqueue the corresponding rawr tiles"""
+    rawr_yaml = cfg.yml.get('rawr')
+    assert rawr_yaml is not None, 'Missing rawr configuration in yaml'
+
+    group_by_zoom = rawr_yaml.get('group-zoom')
+    assert group_by_zoom is not None, 'Missing group-zoom rawr config'
+
+    rawr_queue_name = rawr_yaml.get('queue')
+    assert rawr_queue_name, 'Missing rawr queue'
+    rawr_queue = make_rawr_queue(rawr_queue_name)
+
+    msg_marshall_yaml = cfg.yml.get('message-marshall')
+    assert msg_marshall_yaml, 'Missing message-marshall config'
+    msg_marshaller = make_message_marshaller(msg_marshall_yaml)
```

Comment:
We seem to have this chunk of code for configuring the RAWR stuff in several places now. Is it worth moving that into `config.py`, or perhaps just to a shared function?

Reply:
Good call. I thought it would be annoying because each part would use slightly different overlapping pieces of the rawr config, but several cases had the same boilerplate to set up the rawr enqueuer, and just factoring that out got us some good savings. Updated in 2d7f0f7.